src/controllers/RouteController.ts

```diff
@@ -43,11 +47,29 @@ export class RouteController {
       return {page: "account"};
    }
 
+
    // @Authorized()
    @Get("/pay")
    @Render("index.ejs")
-   GetPay(): unknown {
-      const payments = [{receiver: "johndoe", amount: 384, subject: "Lunch at EWI"},{receiver: "Piet", amount: 112, subject: "Coffee"}];
+   async GetPay(@Req() request: Request): Promise<unknown> {
```

Comment:
It would be clearer to use the @CurrentUser decorator here

Reply:
I tried the current user was undefined

Reply:
Fixed it and pushed the changes